crates/uv-build-backend/src/wheel.rs

```diff
@@ -186,7 +186,7 @@ fn write_wheel(
         wheel_subdir_from_globs(
             source_tree,
             &license_dir,
-            license_files,
+            pyproject_toml.license_files_wheel(),
```

Comment:
You could maybe use a peekable iterator above to avoid re-creating it here? But it doesn't look like iterator construction is especially costly here.

Reply:
I decided to just go for simplicity here